exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/AbstractParquetScanBatchCreator.java

```diff
@@ -214,10 +225,14 @@ protected ScanBatch getBatch(ExecutorFragmentContext context, AbstractParquetRow
         mapWithMaxColumns = createReaderAndImplicitColumns(context, rowGroupScan, oContext, columnExplorer, readers, implicitColumns, mapWithMaxColumns, firstRowGroup, fs,
           firstFooter, true);
       }
+      // do some logging, if relevant
       if ( totalPruneTime > 0 ) {
         logger.info("Finished parquet_runtime_pruning in {} usec. Out of given {} rowgroups, {} were pruned. {}", totalPruneTime, totalRowgroups, rowgroupsPruned,
           totalRowgroups == rowgroupsPruned ? "ALL_PRUNED !!" : "");
       }
+      if ( countMatchClassCastExceptions > 0 ) {
+        logger.info("Run-time pruning check skipped for {} out of {} rowgroups due to: {}",countMatchClassCastExceptions, totalRowgroups, matchCastErrorMessage);
```

Comment:
Minor point: The word 'check' should be removed since at least a partial check was performed and the failure happened during that process.

Reply:
OK, done